packages/core/src/sync/sync-api.ts

```diff
@@ -282,7 +298,8 @@ export class SyncApi implements ISyncApi {
       activeSyncs: historySyncJobs.map((job) => {
         const jobData = job.data as SyncJobData
         return {
-          currentBlock: jobData.fromBlock,
+          currentBlock: jobData.currentBlock || jobData.fromBlock,
+          startBlock: jobData.fromBlock,
```

Comment:
startBlock for the continuousSync below?

Reply:
I'm not sure what the start block would represent in a continuous sync. I would guess the user would want to know how far from the latest block we are which is current block. What does start block mean to you?

Reply:
whatever block the continuous sync started at.  Presumably this will be whatever the current ethereum block is when the node starts up